.eslintrc.js

```diff
@@ -93,14 +93,13 @@ module.exports = {
         'no-null/no-null': 'error',
         '@typescript-eslint/no-empty-function': 'off',
         '@typescript-eslint/no-unused-vars': 'off',
-        // New rules --> New TODOs
+        '@typescript-eslint/no-floating-promises': 'error', // Promises must catch errors or be awaited.
```

Comment:
Question: Should this rule be applied to only non test files? Maybe excluding files with the word `test` or `Test` in them. I don't think there is the `.spec.` pattern to apply to in this codebase.

Reply:
That would add a source of accidents to the tests. What would be gained?

Reply:
@damntrecky here's a recent example where tests were not correctly await'd: https://github.com/aws/aws-toolkit-vscode/pull/4240/files#r1444539552